_layouts/default.html

```diff
@@ -63,6 +63,54 @@
         <script defer src="/js/bootstrap.min.js"></script>
         <script defer src="/js/scripts.js"></script>
 
+        <!-- Schema.org JSON-LD Markup for Search Engine Optimization -->
+        <script type='application/ld+json'>
+        {
+            "@context": "https://schema.org/",
+            "@type": "SoftwareApplication",
+            "applicationCategory": "FinanceApplication",
+            "name": "Bisq",
+            "image": "https://bisq.network/images/bisq-logo.svg",
+            "availableOnDevice": "Desktop",
+            "author": {
+            "@type": "Project",
+            "url": "https://bisq.network/dao/",
+            "name": "Bisq Decentralized Autonomous Organization",
+            "sameAs": [
```

Comment:
Are you sure this `sameAs` block is best placed in the `author` block? Might it be better to put it in the top-level block?

Reply:
It's correct